build/Signing/Microsoft.NET.Signing.proj

```diff
@@ -25,4 +24,4 @@
     <Exec Command="$(SignTool) -nugetPackagesPath &quot;$(NuGet_Packages)&quot; -config &quot;@(SigningConfig)&quot; &quot;$(OutDir.TrimEnd('\'))&quot;"
```

Comment:
@srivatsn items imply collections which isn't possible here.  The tool only accepts a single argument hence a property is more appropriate.